torch_xla/csrc/tensor_methods.h

```diff
@@ -883,6 +883,8 @@ std::tuple<XLATensorPtr, XLATensorPtr> var_mean(const XLATensorPtr& input,
 // Like reshape, but it returns a view into the original tensor.
 XLATensorPtr view(const XLATensorPtr& input,
```

Comment:
Do we keep both `view` and `view_symint` going forward?

Reply:
Yeah.

For view op, I've modified my code to only use `view_symint`.

 However, there're 3 other references in tensor_ops.cpp that uses this function which belongs to other ops. This PR is only intended to fix the `view` ops so I'll leave it there.